vagrant-pxe-harvester/settings.yml

```diff
@@ -0,0 +1,89 @@
+---
+##########################################################################
+# NOTE: this is a YAML file so please pay close attention to the leading #
+# spaces as they are significant.                                        #
+##########################################################################
+
+#
+# harvester_iso_url
+# harvester_kernel_url
+# harvester_initrd_url
+#
+# Harvester media to install. The URL scheme can be either 'http', 'https', or
+# 'file'. If the URL scheme is 'file', the given media will be copied from the
+# local file system instead of downloading from a remote location.
+harvester_iso_url: https://releases.rancher.com/harvester/v0.2.0-rc5/harvester-amd64.iso
+harvester_kernel_url: https://releases.rancher.com/harvester/v0.2.0-rc5/harvester-vmlinuz-amd64
+harvester_ramdisk_url: https://releases.rancher.com/harvester/v0.2.0-rc5/harvester-initrd-amd64
```

Comment:
v0.2.0 is a specific version and the download link should work, can we use it instead, thanks

Reply:
yes, that's implemented in the updated patch.

Reply:
sorry, what do u mean by the updated path?

Reply:
I mean I updated the commit to include the URL implementation as you suggested. The current commit supports http/https scheme for remote download and file scheme for copying media from local file system. e.g.

https://releases.rancher.com/harvester/v0.2.0-rc5/harvester-amd64.iso
file:///mydir/harvester-amd64.iso

Reply:
sure, this is great thanks. What I would suggest here is to use the `v0.2.0` release instead of the `r5` release as the default example. As RC is more used for testing purposes, thanks.

Reply:
I see. Sorry I misunderstood your first comment. Will update it to `v0.2.0`.